README.md

```diff
@@ -1,5 +1,5 @@
 **[Installation](#installation)** |
-**[Documentation](#documentation)** |
+**[Documentation](http://jupyterlab.readthedocs.io/en/latest/)** |
```

Comment:
I think the stable version would be better?

Reply:
I would say remove latest/stable, so that the setting of the RTD project is used.

Reply:
Even better.

Reply:
@mpacer I'm ok with the PR with the change